packages/sak/dekorator/src/DekoratorMedFeilviserSakIndex.tsx

```diff
@@ -64,10 +66,17 @@ export const DekoratorMedFeilviserSakIndex = ({
             {tittel}
           </InternalHeader.Title>
           <Spacer />
+          {nyVersjonTilgjengelig && (
+            <InternalHeader.Button onClick={() => window.location.reload()}>
```

Comment:
Blir Button synlig når man står inne i en åpen sak? Risiko for å miste arbeidet?
Gjør reload at man blir stående med aktuell sak / behandling eller havner man annet sted ? 
Tenker det er nyttig at vi er tydelige dersom man evt mister arbeid.